### PR TITLE
Output annotations for `brew audit`.

### PR DESCRIPTION
**Pull request by reitermarkus**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/HEAD/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).
- [x] Have you successfully run `brew style` with your changes locally?
- [x] Have you successfully run `brew tests` with your changes locally?

### CODE REVIEW

**Comment by SMillerDev** [comment]
What would this do/add/fix?

**Comment by reitermarkus** [comment]
Add annotations in the “Files” tab in the GitHub UI.

**Comment by SMillerDev** [comment]
Can't really judge the code. But the feature sounds great.

File: Library/Homebrew/dev-cmd/audit.rb [hunk 1]
```diff
@@ -153,6 +153,15 @@ def audit
       else
         puts "#{f.full_name}:", problem_lines.map { |s| "  #{s}" }
       end
+
+      next unless ENV["GITHUB_ACTIONS"]
```
Comment (MikeMcQuaid) [inline Library/Homebrew/dev-cmd/audit.rb:157]: Might be nice to have a `GitHub::Actions.something?` boolean method for this.

**Review by MikeMcQuaid — APPROVED** [review]
Nice work, thanks for this!

File: Library/Homebrew/dev-cmd/audit.rb (hunk 1 above)
Comment (reitermarkus) [reply Library/Homebrew/dev-cmd/audit.rb:157]: Will keep this in mind, I don't think it's worth adding this yet. Might be possible to come up with a nicer API in future like

```ruby
Github::Actions.annotate do
  error "message", …
end
```

**Pull request merged**